File: FitnessWeb.Frontend/AngularApp/src/commands/updateFitnessExerciseCommand.ts
export class UpdateFitnessExerciseCommand {
    id: number;
    name: string;
    countOfRepeats: number;
    videoUrl: string;

    constructor(id: number, name: string, countOfRepeats: number, videoUrl: string) {
        this.id = id;
        this.name = name;
        this.countOfRepeats = countOfRepeats;
        this.videoUrl = videoUrl;
    }
}

File: FitnessWeb.API/Commands/UpdateFitnessExerciseCommand.cs
using MediatR;

namespace FitnessWeb.API.Commands
{
    public class UpdateFitnessExerciseCommand : IRequest
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public int CountOfRepeats { get; set; }
        public string VideoUrl { get; set; }
    }
}
